application/app/models/zenodo/search_response.rb

```diff
@@ -0,0 +1,14 @@
+module Zenodo
+  class SearchResponse
```

Comment:
I wonder if it could make sense to include the ActsAsPage mixin to provide more pagination functionality

Reply:
It does - will do in the next iteration